docker/docker-compose.yml

```diff
@@ -461,6 +461,9 @@ services:
     environment:
       MF_UI_PORT: ${MF_UI_PORT}
       MF_UI_MQTT_WS_URL: ${MF_UI_MQTT_WS_URL}
+      PS_SID: ${PS_SID}
+      PS_GROUP_KEY: ${PS_GROUP_KEY}
+      
```

Comment:
Could you put values on `.env` file for those env variables??

Reply:
I shouldn't commit the values for those. Each user that downloads and spins their own Orb copy, can, in fact, sign up for IronClad and then add their own keys.

Staging environment contains the keys for testing, while live environment should be built with definitive keys for this specific environment.

Hence, the values are commited in orb.live's repo, but not available in orb's repo.

The correct procedure, either via .env file(in case UI is able to be built via docker-compose), or environment vars, is explained above.

I suspect settings those to .env won't take effect, since we need those values at the moment the image is built - and orb-ui is compiled.